apiserver/facades/client/spaces/spaces_rename.go

```diff
@@ -0,0 +1,228 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package spaces
+
+import (
+	"github.com/juju/errors"
+	"gopkg.in/juju/names.v3"
+	"gopkg.in/mgo.v2/txn"
+
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/params"
+	jujucontroller "github.com/juju/juju/controller"
+	"github.com/juju/juju/core/constraints"
+	"github.com/juju/juju/core/permission"
+	"github.com/juju/juju/core/settings"
+	"github.com/juju/juju/state"
+)
+
+// RenameSpaceModelOp describes a model operation for renaming a space.
+type RenameSpaceModelOp interface {
```

Comment:
Weren't we going to just use the `state.ModelOperation` without the outer interface?

Reply:
Sorry! I thought I had deleted that, I was pretty sure..